apis/v1beta1/validation/httproute_test.go

```diff
@@ -1176,3 +1176,67 @@ func TestValidateRequestRedirectFiltersWithNoBackendRef(t *testing.T) {
 		})
 	}
 }
+
+func toDuration(durationString string) *gatewayv1b1.Duration {
+	return (*gatewayv1b1.Duration)(&durationString)
+}
+
+func TestValidateHTTPTimeouts(t *testing.T) {
```

Comment:
Now that Gateway API supports CEL validation, you need to add a similar CEL validation test function (i.e., `func TestHTTPRouteTimeouts`) in here:  https://github.com/kubernetes-sigs/gateway-api/blob/main/pkg/test/cel/httproute_test.go

Reply:
+1 Would be great to have good coverage for our CEL validation tests. Note that these currently require a cluster with the latest CRDs installed. You can take a look at https://github.com/kubernetes-sigs/gateway-api/blob/main/hack/verify-crds-kind.sh for an example of how we run these tests.

Reply:
addressed in e148233a0a32ebdd63d5d8c52468f0910ef345a2